python/lsst/cell_coadds/_single_cell_coadd.py

```diff
@@ -131,6 +136,32 @@ def common(self) -> CommonComponents:
         # Docstring inherited.
         return self._common
 
+    @property
+    def aperture_correction_map(self) -> SingleCellCoaddApCorrMap | None:
+        """Mapping of algorithm name to aperture correction values.
+
+        Returns
+        -------
+        aperture_correction_map : `frozendict` [`str`, float] | None
+            Mapping of algorithm name to aperture correction values.
+            If None, no aperture correction is applied.
+        """
+        return self._aperture_correction_map
+
+    @property
+    def aperture_corrected_algorithms(self) -> tuple[str, ...]:
+        """An iterable of algorithm names that have aperture correction values.
+
+        Returns
+        -------
+        aperture_corrected_algorithms : `tuple` [`str`, ...]
+            List of algorithms that have aperture correction values.
+        """
+        if self._aperture_correction_map:
+            return tuple(self._aperture_correction_map.keys())
```

Comment:
I don't think there's any need to wrap this in `tuple`.  Just return `keys()` and annotate as `collections.abc.Set`.